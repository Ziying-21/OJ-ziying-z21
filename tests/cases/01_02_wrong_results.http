###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 377
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:40 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:40.647Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 433
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:41 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":296727},{"id":1,"info":"","memory":0,"result":"Wrong Answer","time":282901}],"created_time":"2022-09-09T02:18:40.647Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0},"updated_time":"2022-09-09T02:18:41.232Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 375
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:41 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:41.659Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 433
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:42 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":162935},{"id":1,"info":"","memory":0,"result":"Runtime Error","time":360232}],"created_time":"2022-09-09T02:18:41.659Z","id":1,"result":"Runtime Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:18:42.186Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 379
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:42 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:42.674Z","id":2,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/2 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 427
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:43 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Error","time":49057},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:42.674Z","id":2,"result":"Compilation Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0},"updated_time":"2022-09-09T02:18:42.727Z"}
