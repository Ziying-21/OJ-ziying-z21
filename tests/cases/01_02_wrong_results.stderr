[2022-09-09T02:18:39Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:18:39Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:18:40Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000162
[2022-09-09T02:18:40Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 377 "-" "-" 0.000490
[2022-09-09T02:18:41Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 433 "-" "-" 0.000315
[2022-09-09T02:18:41Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 375 "-" "-" 0.001712
thread 'main' panicked at 'Hello World!', temp_1/main.rs:1:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[2022-09-09T02:18:42Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 433 "-" "-" 0.000511
[2022-09-09T02:18:42Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 379 "-" "-" 0.001650
error: 1 positional argument in format string, but no arguments were given
 --> temp_2/main.rs:1:35
  |
1 | fn main() { println!("Hello World!{}"); }
  |                                   ^^

error: aborting due to previous error

[2022-09-09T02:18:43Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/2 HTTP/1.1" 200 427 "-" "-" 0.000340
